### DISCUSSION TITLE
D-link DUB-H7/E1 slim doesn' t work !

### DISCUSSION THREAD

**Discussion by CpServiceSpb** [mvp/uhubctl#544] [General]
D-link DUB-H7/E1 slim doesn' t work - VBUS is not cut.
Tests were made by USB tester with OLED !
One thing - charging port is off and the on and -r switch doesn' t matter.

**Comment by mvp** [comment]
Do you suggest that we should remove D-link DUB-H7 rev E from supported table?

**Comment by CpServiceSpb** [comment]
Unfortunately I can not understand there are 2 different types of E
revision - one as B, C, D - "fat" and other is "slim".

пн, 20 нояб. 2023 г. в 02:33, mvp ***@***.***>:

**Comment by mvp** [comment]
I am confused now. I assume you are saying that this hub rev E1 doesn't work: https://www.regard.ru/product/347156/usb-koncentrator-d-link-dub-h7-e1
but this one rev E does work https://www.amazon.com/dp/B0002AFZVM?

**Reply by CpServiceSpb** [reply]
Regarding [slim E1](https://www.regard.ru/product/347156/usb-koncentrator-d-link-dub-h7-e1), yes, it is indeed. It doesn' t work !
Regarding [`ordinal E1]( https://www.amazon.com/dp/B0002AFZVM), I didn' t even see and touch it at all.
